server/tilesource/tiff_reader.py

```diff
@@ -139,7 +139,7 @@ def _validate(self):
 
         :raises: ValidationTiffException
         """
-        if self._tiffFile.GetField('SamplesPerPixel') != 3:
+        if self._tiffFile.GetField('SamplesPerPixel') < 3:
```

Comment:
Do we have an RGBA JPEG image to test with? I want to make sure that viewers handle them sanely before allowing the server to send one.